ldpd/lde.h

```diff
@@ -114,6 +114,8 @@ struct fec_nh {
 };
 #define F_FEC_NH_NEW		0x01
 #define F_FEC_NH_CONNECTED	0x02
+#define F_FEC_NH_DEFER		0x04		/* running ordered control */
```

Comment:
please line up the defines with those originally around it.

Reply:
done